### DISCUSSION TITLE
post/iac-explore-ansible-and-terraform/

### DISCUSSION THREAD

**Discussion by giscus** [lyc8503/lyc8503#24] [Announcements]
# post/iac-explore-ansible-and-terraform/

很久以前就听说过了 基础设施即代码 (Infrastructure-as-Code, 缩写 IaC) 的做法, 最近正好有不错的机会实践了一下, 写此文以记录. 背景我现有的基础设施主要是我的 HomeLab 和阿里云的一些 Serverless 服务, 两者之前都有相关的博客提到过, 现在都处于稳定运行状态, 相对不需要大的变动. 不过我还有些必要的辅助服务需要在海外 VPS 上运行, 主要出于

https://blog.lyc8503.net/post/iac-explore-ansible-and-terraform/

**Comment by luckyops** [comment]
好耶，学到了Terraform，之前一直用ansible，Terraform 还没搞过，嘿嘿。
